Phoenix/Exceptions/UserNotFoundException.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Phoenix.Exceptions
+{
+    /// <summary>
+    /// Exception to be thrown by services when a user we expect to find is not found.
+    /// </summary>
+    public class UserNotFoundException : Exception
+    {
+        private string user = "";
+
+        public UserNotFoundException(string usr)
+            :base(string.Format("The user {0} was not found.", usr))
+        {
+            user = usr;
+        }
+
+        public UserNotFoundException(string usr, Exception innerException)
+            : base(string.Format("The user {0} was not found.", usr), innerException)
+        {
+            user = usr;
+        }
+
+        public string NotFoundUser()
+        {
+            return user;
+        }
+    }
```

Comment:
I think the benefit is the naming? So when I throw an exception, its a bit clearer what the problem is. 
If it is more confusing than helpful, I'll get rid of them. I was going for making exception throwing easy to read :)